packages/appstore/src/components/appstore-banner/appstore-banner.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import SVGMigrationBanner from './svg-migration-banner';
+
+const AppstoreBanner = () => {
+    //TODO: will add proper condition check when API is ready
+    if (true) return <SVGMigrationBanner />;
```

Comment:
why do we need a separate component `SVGMigrationBanner`?
can we move the content of `SVGMigrationBanner` to `AppstoreBanner`?

Reply:
We need this because later on for wallets they have their own banner also. I do this just to make it easier if there's more banner coming in afterwards 😁

Reply:
oh, ok.. great 👍🏼